iso15118/shared/pki/configs/contractLeafCert.cnf

```diff
@@ -12,4 +12,4 @@ domainComponent			= MO
 basicConstraints		= critical,CA:false
 keyUsage				= critical,digitalSignature,nonRepudiation,keyEncipherment,keyAgreement
 subjectKeyIdentifier	= hash
-
+authorityInfoAccess = OCSP;URI:http://ca.myown.com/, caIssuers;URI:http://ca.myown.com/Intermediate-CA.cer
```

Comment:
Like example.com, this seems not to be a real site -- is there a benefit to replacing the canonical fake site with one where this fact is less obvious?

Reply:
no, not really, was just the example I extracted... I can set it back to example

Reply:
Probably better -- `ca.myown.com` isn't currently registered, but `myown.com` redirects to oprah.com, the site of Oprah Winfrey, an American talk show host.